api/_utils.ts

```diff
@@ -302,10 +302,15 @@ export const queries: Record<number, () => QueryBase> = {
  * @returns An instance of the `RelayFeeCalculator` for the specific chain specified by `destinationChainId`
  */
 export const getRelayerFeeCalculator = (destinationChainId: number) => {
+  const queryFn = queries[destinationChainId];
+  if (typeof queryFn === undefined) {
```

Comment:
Is this working as intended? typeof produces a string, so won't this always evaluate false?

Separately, I did something similar and was encouraged to use:

```suggestion
  if (queryFn === undefined) {
```

Reply:
Ah yes, great point. You're 100% right.